src/liblvgl/display.c

```diff
@@ -117,7 +117,12 @@ void display_initialize(void) {
 		printf("Error initializing input driver\n");
 	}
 
-	// lv_theme_set_current(lv_theme_alien_init(40, NULL));
+	lv_theme_t *th = lv_theme_default_init(lv_disp_get_default(),
```

Comment:
you can set LVGL default theme without writing code.

![image](https://github.com/purduesigbots/liblvgl/assets/12783829/7f6193e5-c0b0-40e7-a4bb-c1e63695e0a4)

![image](https://github.com/purduesigbots/liblvgl/assets/12783829/a7f0a5d6-d570-485b-90ab-2002eccca6dd)